app/modules.go

```diff
@@ -221,7 +221,7 @@ func orderBeginBlockers(allModuleNames []string) []string {
 	// Epochs must come before staking, because txfees epoch hook sends fees to the auth "fee collector"
 	// module account, which is then distributed to stakers. If staking comes before epochs, then the
 	// funds will not be distributed to stakers as expected.
-	ord.FirstElements(upgradetypes.ModuleName, epochstypes.ModuleName, capabilitytypes.ModuleName)
```

Comment:
Did it say to do this somewhere?

Reply:
https://github.com/cosmos/cosmos-sdk/blob/main/UPGRADING.md#set-preblocker

Yeah it did, but it's failing e2e, I think we may need to keep it the way it is :confused: